src/server/config/schema.js

```diff
@@ -52,6 +52,7 @@ module.exports = () => Joi.object({
       disableProtection: Joi.boolean().default(false),
       token: Joi.string().optional().notes('Deprecated')
     }).default(),
+    staticDirectoryListings: Joi.boolean().default(Joi.ref('$dev')),
```

Comment:
@epixa if the config doesn't work then we have a much bigger issue... All together I'm giving up on trying to keep it in.